src/test/java/core/basesyntax/serviceimpl/ParserServiceImplTest.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.serviceimpl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.ParserService;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ParserServiceImplTest {
+    private static ParserService parserService;
+
+    @BeforeAll
+    static void init() {
+        parserService = new ParserServiceImpl();
+    }
+
+    @Test
+    void when_NegativeAmount_NotOk() {
```

Comment:
which method is being tested here - not an informative method name. The same in other tests